### PR TITLE
Kills Account page and updates logout flow

### PR DESCRIPTION
**Pull request by rosschapman**
Home page:
percy___automated_visual_reviews_for_web_apps

Other page:
my_organization_0_-_percy___automated_visual_reviews_for_web_apps

### CODE REVIEW

File: app/styles/components/_user_menu.sass [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.UserMenu ul
```
Comment (fotinakis) [inline app/styles/components/_user_menu.sass]: Optional: for consistency with the other sass files, have a top-level `.UserMenu` and nest `ul` and `li` blocks underneath.

File: app/templates/components/user-menu.hbs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{{user-avatar user=user width=36}}
+{{#if showMenu}}
+  <ul>
+    <li {{action "invalidateSession"}}>Logout {{fa-icon "sign-out" classNames="fa--left"}}</li>
```
Comment (fotinakis) [inline app/templates/components/user-menu.hbs]: Yeah drop the sign-out icon, it looks funny. :)

File: tests/acceptance/login-test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import setupAcceptance, {setupSession} from '../helpers/setup-acceptance';
 
-describe('Acceptance: Login', function() {
+describe('Acceptance: Login and Logout', function() {
```
Comment (fotinakis) [inline tests/acceptance/login-test.js]: Maybe just leave as 'Login' which is more accurate.

**Comment by fotinakis** [comment]
Seeing this a lot in dev: 
screen shot 2017-03-13 at 2 42 17 pm

**Comment by rosschapman** [comment]
@fotinakis see screenshots for updated styling.

**Pull request merged**